Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.7
```

Comment:
Just something we should do later: https://github.com/openshift/openshift-state-metrics/issues/3